data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -216,14 +225,11 @@ then
 	cp manifests/* /etc/mcc/bootstrap/
 	cp auth/kubeconfig-kubelet /etc/mcs/kubeconfig
 	cp mco-bootstrap/bootstrap/machineconfigoperator-bootstrap-pod.yaml /etc/kubernetes/manifests/
-        if [ -d mco-bootstrap/baremetal/manifests ]; then
-            cp mco-bootstrap/baremetal/manifests/* /etc/kubernetes/manifests/
-            cp --recursive mco-bootstrap/baremetal/static-pod-resources/* /etc/kubernetes/static-pod-resources/
-        fi
-	if [ -d mco-bootstrap/openstack/manifests ]; then
-		cp mco-bootstrap/openstack/manifests/* /etc/kubernetes/manifests/
-		cp --recursive mco-bootstrap/openstack/static-pod-resources/* /etc/kubernetes/static-pod-resources/
-	fi
+
+  copy_static_resources_for baremetal
```

Comment:
the indent doesn't line up

Reply:
it would be nice to fix this.

Reply:
I just now realized my Golang IDE hides this from me because of the mixture of tabs and whitespace .
Fixed.